mittab/libs/data_import/import_teams.py

```diff
@@ -72,98 +91,100 @@ def import_teams(fileToImport):
                 continue
             team_school = School.objects.get(name__iexact=school_name)
 
-        team_seed = sh.cell(i,2).value.strip().lower()
-        if team_seed == 'full seed' or team_seed == 'full':
-            team_seed = 3
-        elif team_seed == 'half seed' or team_seed == 'half':
-            team_seed = 2
-        elif team_seed == 'free seed' or team_seed == 'free':
-            team_seed = 1
-        elif team_seed == 'unseeded' or team_seed == 'un' or team_seed == 'none' or team_seed == '':
-            team_seed = 0
-        else:
-            team_errors.append(team_name + ': Invalid Seed Value')
-            continue
-
-        deb1_name = sh.cell(i,3).value
-        if deb1_name == '':
-            team_errors.append(team_name + ': Empty Debater-1 Name')
-            continue
-        try:
-            Debater.objects.get(name=deb1_name)
-            team_errors.append(team_name + ': Duplicate Debater-1 Name')
-            continue
-        except:
-            pass
-        deb1_status = sh.cell(i,4).value.lower()
-        if deb1_status == 'novice' or deb1_status == 'nov' or deb1_status == 'n':
-            deb1_status = 1
-        else:
-            deb1_status = 0
-        deb1_phone = sh.cell(i,5).value
-        deb1_provider = sh.cell(i,6).value
-
-
-        iron_man = False
-        deb2_name = sh.cell(i,7).value
-        if deb2_name == '':
-            iron_man = True
-        if (not iron_man):
+        # check seeds
+        team_seed, changed_seed = _create_seed(team_name=team_name, seed=sh.cell(i, 2).value.strip().lower())
+        if changed_seed:
+            team_errors.append('Changed ' + team_name + ' from "' + sh.cell(i, 2).value.strip().lower()
+                               + '" to unseeded. Note and confirm with school.')
+
+        deb1_name = sh.cell(i, 3).value.strip()
+        deb1_status = _create_status(sh.cell(i, 4).value.lower())
+        deb1_phone = sh.cell(i, 5).value.strip()
+        deb1_provider = sh.cell(i, 6).value.strip()
+        deb1, deb1_created = Debater.objects.get_or_create(name=deb1_name, novice_status=deb1_status, phone=deb1_phone,
```

Comment:
Same here about not updating via excel import. Haven't heard any complaints about the current behavior so I'd prefer not to change things

Reply:
In just-made changes, I made overwrites contingent on a passed parameter in all import scripts. Theoretically, some extension could be given where a user could choose, although I'll leave that as an "exercise for the reader".